v2/googlecloud-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/templates/TextIOToBigQuery.java

```diff
@@ -263,8 +262,10 @@ public interface Options
         regexes = {"[a-zA-Z0-9_]+"},
         description = "UDF Javascript Function Name",
         helpText =
-            "The name of the function to call from your JavaScript file. Use only letters, digits,"
-                + " and underscores.",
+            "The name of the JavaScript user-defined function (UDF) that you want to use. For example, if " +
+                    "your JavaScript function code is `myTransform(inJson) { /*...do stuff...*/ }`, then " +
+                    "the function name is `myTransform`. For sample JavaScript UDFs, " +
+                    "see UDF Examples(https://github.com/GoogleCloudPlatform/DataflowTemplates#udf-examples)",
```

Comment:
see https://github.com/GoogleCloudPlatform/DataflowTemplates#udf-examples.

Reply:
We have agreed to use below format for hyper links  : 

UDF Examples (https://github.com/GoogleCloudPlatform/DataflowTemplates#udf-examples)

@rajkgupt Please add a space between Examples and (link).

Reply:
done